### PR TITLE
Fix dead link in management-console.html

### PR DESCRIPTION
**Pull request by geertschuring**
There was a dead link to Jolokia.org which this commit fixes.

### CODE REVIEW

File: src/components/artemis/documentation/latest/management-console.html [hunk 1]
```diff
@@ -226,4 +226,4 @@ <h3 id="status-logging"><a class="anchor" href="#status-logging"></a><a class="l
 </div>
 </div>
 </body>
-</html>
```
Comment (geertschuring) [inline src/components/artemis/documentation/latest/management-console.html:229]: This is an unintended whitespace change. Please ignore/undo.

Comment (jbertram) [reply src/components/artemis/documentation/latest/management-console.html:229]: Can you fix the commit and do a `push -f`?

**Comment by jbertram** [comment]
To be clear, this documentation is generated from AsciiDoc in the broker's repository. I've [fixed this link there](https://github.com/apache/activemq-artemis/commit/a60a8a12cfad317b314031fcde3b9a87c7704d74). I will merge this PR once you fix your unintended change.

**Comment by jbertram** [comment]
This will be fixed upon the next release of Artemis.

**Pull request closed without merging**